code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -755,3 +755,8 @@
 	icon_state = "lean"
 	list_reagents = list(/datum/reagent/consumable/lean = 20)
 	random_sprite = FALSE
+
+/obj/item/reagent_containers/food/drinks/colocup/lean/suicide_act(mob/living/carbon/user)
+	user.visible_message(span_suicide("[user] begins to chug the cup of lean and is starting to turn purple! It looks like [user.p_theyre()] trying to commit suicide!"))
```

Comment:
is it gonna give message only to user, or to everyone else?

Reply:
i belive it will give a message to everyone else
but i am lonley on local and can not test it.

Reply:
yea it's fine, me just stupid, you can close this comment